targets/targets.json

```diff
@@ -12155,6 +12155,60 @@
         "mbed_ram_size": "0x40000",
         "release_versions": [ "5" ]
     },
+    "DISCO_L562QE": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M33FE",
+        "extra_labels_add": [
+            "STM32L5",
+            "STM32L562xx",
+            "MX25LM51245G"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "STMOD",
+            "PMOD"
+        ],
+        "components_add": [
+            "FLASHIAP"
+        ],
+        "macros_add": [
+            "STM32L562xx",
+            "MBED_TICKLESS",
+            "EXTRA_IDLE_STACK_REQUIRED"
+        ],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC (need HW patch) | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI | USE_PLL_MSI",
+                "value": "USE_PLL_MSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "stmod_sel": {
+                "help": "1 for UART / 0 for SPI - see board User Manual, STMOD_1/2/3/4 are configurable",
+                "value": 1
+            },
+            "lpticker_lptim": {
+                "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
+                "value": 1
+            }
+        },
+        "overrides": { "lpticker_delay_ticks": 0 },
```

Comment:
please fix formatting as the rest of this json (new line should be here?)

Reply:
done